assets/js/icons/library/arrow-down-alt2.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { SVG } from 'wordpress-components';
+
+const arrowDownAlt2 = ( { className, size, ...extraProps } ) => {
+	const iconClass = [ 'dashicon', 'dashicons-arrow-down-alt2', className ]
+		.filter( Boolean )
+		.join( ' ' );
+	return (
+		<SVG
+			xmlns="http://www.w3.org/2000/svg"
+			viewBox="0 0 20 20"
```

Comment:
not sure how will the icon render, but if we want to keep with Gutenberg new grid system, this should be changed to `"-2 -2 24 24"`, but if the icon renders fine and sharp, we should be okay.

Reply:
Icon seems to be rendering fine, this was copied and pasted directly from the existing dashicons implementation. Since this is mostly temporary (we can replace once we can bump component versions), not sure if we need to change.